content/en/docs/reference/config-api/apiserver-tracingconfiguration.v1alpha1.md

```diff
@@ -0,0 +1,8 @@
+---
+title: TracingConfiguration (v1alpha1)
+content_type: tool-reference
+package: apiserver.config.k8s.io/v1alpha1
+auto_generated: true
+---
```

Comment:
This looks wrong - why is the page blank?

I'd expect to have the code in https://github.com/kubernetes-sigs/reference-docs/ generate the content, and have that generated content added as its own commit.

That generation is only a nice-to-have. If it's not yet done, omit `content/en/docs/reference/config-api/apiserver-tracingconfiguration.v1alpha1.md`, don't modify  `content/en/docs/reference/_index.md`, and don't link to that config API reference.